src/drawer/index.tsx

```diff
@@ -105,34 +83,36 @@ export default class Drawer extends PureComponent <DrawerProps, DrawerState> {
 
   scrollLock = () => {
     if (canUseDOM) {
-      document.documentElement.classList.add('scroll-lock')
+      document.documentElement.classList.add('kirk-scroll-lock')
```

Comment:
We are using a polyfill on the SPA, classList doesn't seem to be supported by our audience

Reply:
as discussed IRL, classList seems well supported.